UI/basicGUI/src/pages/home/home.ts

```diff
@@ -111,8 +115,9 @@ loadLegend(){
     var weatherMarker = this.weatherMarker
 
     var onClickSetBlock = this.onClickSetBlock
-    var getRouteWithBlock = this.getRouteWithBlock
+    // var getRouteWithBlock = this.getRouteWithBlock
```

Comment:
Please remove it off!

Reply:
Actually, the getRouteWithBlock was defined twice in this function